### PR TITLE
File pattern changes needed for lite-xl#1322

### PR DESCRIPTION
**Pull request by jgmdev**
As explained on title these are all the changes needed for PR https://github.com/lite-xl/lite-xl/pull/1322

Missing is contacting or PR change to https://github.com/FilBot3/lite-xl-language-containerfile to adjust https://github.com/FilBot3/lite-xl-language-containerfile/blob/main/init.lua

### CODE REVIEW

File: plugins/language_caddyfile.lua [hunk 1]
```diff
@@ -2,7 +2,7 @@
 local syntax = require "core.syntax"
 
 syntax.add {
-  files = { "Caddyfile" },
+  files = { "/Caddyfile" },
```
Comment (Guldoman) [inline plugins/language_caddyfile.lua:5]: Should maybe add `$`?

File: plugins/language_fstab.lua [hunk 2]
```diff
@@ -4,7 +4,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "fstab",
-  files = { "fstab" },
+  files = { "/fstab" },
```
Comment (Guldoman) [inline plugins/language_fstab.lua:7]: `$`

File: plugins/language_make.lua [hunk 3]
```diff
@@ -3,7 +3,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "Makefile",
-  files = { "Makefile", "makefile", "%.mk$" },
+  files = { "/Makefile", "/makefile", "%.mk$" },
```
Comment (Guldoman) [inline plugins/language_make.lua:6]: `$`

**Review by Guldoman — COMMENTED** [review]
We should probably bump the plugins versions too.

File: plugins/language_caddyfile.lua (hunk 1 above)
Comment (jgmdev) [reply plugins/language_caddyfile.lua]: good catch, done on all the ones you spotted

**Comment by jgmdev** [comment]
> We should probably bump the plugins versions too.

true, totally forgot that, done, thank you!

**Comment by Guldoman** [comment]
Should we wait for the next release before merging this, or are the changes not that breaking?

**Comment by jgmdev** [comment]
before merging this we should first merge the change on https://github.com/lite-xl/lite-xl/pull/1322 and then make a 2.1.2 release

**Pull request closed without merging**